src/components/app.tsx

```diff
@@ -9,19 +9,55 @@ import { MainContentComponent } from "./main-content";
 import { BottomBarComponent } from "./bottom-bar";
 
 interface IProps extends IBaseProps {}
-interface IState {}
+interface IState {
+  width: number;
+  height: number;
+}
+const kAppWidth = 960;
+const kAppHeight = 850; // todo: should be 618
 
 @inject("stores")
 @observer
 export class AppComponent extends BaseComponent<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+    this.state = {
+      width: 0,
+      height: 0
+    };
+    this.updateWindowDimensions = this.updateWindowDimensions.bind(this);
+  }
+
+  public componentDidMount() {
+    this.updateWindowDimensions();
+    window.addEventListener("resize", this.updateWindowDimensions);
```

Comment:
Looks like a straightforward way to handle scaling and it works for me. @sfentress IIRC you implemented scaling for the ConnectedBio app, did you do anything more involved than this? Was `React SizeMe` relevant?

Reply:
I also tried `react-aspect-ratio`, but it didn't work for what we needed.  I'm not crazy about adding CSS this way, but it works.